### PR TITLE
Move video shortcodes to shortcodes page

### PR DESCRIPTION
**Pull request by jdbaldry**
Closes https://github.com/grafana/writers-toolkit/issues/511

### CODE REVIEW

File: docs/sources/write/shortcodes/index.md [hunk 1]
```diff
@@ -407,6 +407,69 @@ Produces:
...
+[dashboards]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/dashboards"
+[dashboards]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/dashboards"
+{{%/* /docs/reference */%}}
+```
+
+The content within the shortcode tags is as follows:
+
+`[LABEL]: "PROJECT PATH PREFIX -> REFERENCE"`
+
+- _`LABEL`_ - The label you'll use in the reference-style links in the file.
```
Comment (chri2547) [inline docs/sources/write/shortcodes/index.md:433]: @jdbaldry , why does the text in lines 433-442 appear blue?

File: docs/sources/write/shortcodes/index.md [hunk 2]
```diff
@@ -407,6 +407,69 @@ Produces:
...
+  To learn about version substitution, refer to [About version substitution](#about-version-substitution).
+  Don't include trailing slashes in the path.
+
+Then add the link in the body of the file in the following format:
+
+```markdown
+For more information about Grafana dashboards, refer to [Dashboards][dashboards].
+```
+
+- If the page you're on is `/docs/grafana-cloud/alerting/`, the returned link is `/docs/grafana-cloud/dashboards/`.
```
Comment (chri2547) [inline docs/sources/write/shortcodes/index.md:451]: same here

**Review by chri2547 — COMMENTED** [review]
Question about '.

File: docs/sources/write/shortcodes/index.md (hunk 1 above)
Comment (jdbaldry) [reply docs/sources/write/shortcodes/index.md:433]: You mean in GitHub? I don't see blue in the local preview. I guess something is confusing the GitHub syntax highlighter.

Comment (chri2547) [reply docs/sources/write/shortcodes/index.md:433]: Yeah, in GH. Ok to ignore then?

**Review by chri2547 — APPROVED** [review]
lgtm!

File: docs/sources/write/shortcodes/index.md (hunk 1 above)
Comment (jdbaldry) [reply docs/sources/write/shortcodes/index.md:433]: Screenshot 2024-03-06 at 14 39 19

Yeah, it looks fine locally

**Pull request merged**